### PR TITLE
Setup ordering structure

### PR DESCRIPTION
**Pull request by fturizo**
The present changes have been implemented to set up how the navigational order of the content on each documentation site should look. To reiterate, the order of navigational elements will be determined by the `generate-nav.py` script according to this simple algorithm:

1. Scan the current directory, and if there's an `Overview.adoc` file in there, it should go first.
2. Afterwards, determine all elements that have an ordinal position assigned to them:
- If the element is a file, its position is set using the `:ordinal:N` Antora attribute as described in https://docs.antora.org/antora/latest/page/page-attributes/
- If the element is a directory, its position is set by having a `.ordN` empty file inside it.
3. Add the elements from low to high according to their numbers (N). If two or more elements have the same ordinal value, then add them in lexicographic order based on the element's name.
4. If a directory's ordinal is overridden in an edition folder (`community`/`enterprise`), then the ordinal value of the element in there will take precedence.
5. Add all remaining elements without ordinals in lexicographic order.

### CODE REVIEW

File: docs/modules/ROOT/pages/Technical Documentation/Payara Micro Documentation/Payara Micro Configuration and Management/Micro Management/HTTP(S) Auto-Binding.adoc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [[http-and-https-auto-binding]]
-= HTTP and HTTPS Auto-Binding
+= HTTP and HTTPS Auto-Binding3
```
Comment (JamesHillyard) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Micro Documentation/Payara Micro Configuration and Management/Micro Management/HTTP(S) Auto-Binding.adoc:2]:
```suggestion
= HTTP and HTTPS Auto-Binding
```

**Review by JamesHillyard — CHANGES_REQUESTED** [review]
Looks good,

Technical Documentation -> Ecosystem -> Overview now has some broken links which need updating

**Comment by JamesHillyard** [comment]
Nav will also need updating as a few filenames have changed

**Comment by fturizo** [comment]
> Nav will also need updating as a few filenames have changed

Nav script changes being worked by @AlanRoth should grab all of these changes.

**Comment by fturizo** [comment]
Refactored navigation script integrated from https://github.com/fturizo/Payara-Documentation/pull/1

**Comment by fturizo** [comment]
@JamesHillyard, can you resolve your previous comments so I can proceed to merge this PR?

**Review by JamesHillyard — APPROVED** [review]

**Comment by AlanRoth** [comment]
Payara 6 master will need the ordinals introduced also

**Comment by fturizo** [comment]
To answer your question, @AlanRoth, I think the safest route would be to handle this manually for other branches since it shouldn't be that difficult. See https://payara.atlassian.net/browse/SERV-835 for a follow-up discussion.

**Pull request merged**